src/libraries/System.Net.Sockets/tests/FunctionalTests/AssemblyInfo.cs

```diff
@@ -4,5 +4,5 @@
 
 using Xunit;
 
-[assembly: SkipOnCoreClr("System.Net.Tests are flaky and/or long running: https://github.com/dotnet/runtime/issues/131")]
+[assembly: SkipOnCoreClr("System.Net.Tests are flaky and/or long running: https://github.com/dotnet/runtime/issues/131", RuntimeTestModes.CheckedRuntime)]
 [assembly: SkipOnMono("System.Net.Tests are flaky and/or long running: https://github.com/dotnet/runtime/issues/131")]
```

Comment:
Do we know if mono actually had problems with these or just copied the coreclr attribution thinking it was about all configs and not just stress?

Reply:
My assumption here is that mono did have problems with this, and `ActiveIssue` isn't usable on an assembly level. @akoeplinger ?

Reply:
I think I just copied from CoreCLR so this might not be needed.